src/App.tsx

```diff
@@ -1,61 +1,35 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo, TodoWithUser } from './types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
+import { FormAddTodo } from './components/FormAddTodo/FormAddTodo';
+import { findUserById } from './utils/findUserById';
 
-export const App = () => {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <form action="/api/todos" method="POST">
-        <div className="field">
-          <input type="text" data-cy="titleInput" />
-          <span className="error">Please enter a title</span>
-        </div>
-
-        <div className="field">
-          <select data-cy="userSelect">
-            <option value="0" disabled>Choose a user</option>
-          </select>
-
-          <span className="error">Please choose a user</span>
-        </div>
-
-        <button type="submit" data-cy="submitButton">
-          Add
-        </button>
-      </form>
+function preparedTodos(todos: Todo[]) {
+  return todos.map((todo: Todo) => ({
+    ...todo,
+    user: findUserById(usersFromServer, todo.userId),
+  }));
+}
 
-      <section className="TodoList">
-        <article data-id="1" className="TodoInfo TodoInfo--completed">
-          <h2 className="TodoInfo__title">
-            delectus aut autem
-          </h2>
-
-          <a className="UserInfo" href="mailto:Sincere@april.biz">
-            Leanne Graham
-          </a>
-        </article>
-
-        <article data-id="15" className="TodoInfo TodoInfo--completed">
-          <h2 className="TodoInfo__title">delectus aut autem</h2>
-
-          <a className="UserInfo" href="mailto:Sincere@april.biz">
-            Leanne Graham
-          </a>
-        </article>
+export const App = () => {
+  const [todos, setTodos] = useState(preparedTodos(todosFromServer));
 
-        <article data-id="2" className="TodoInfo">
-          <h2 className="TodoInfo__title">
-            quis ut nam facilis et officia qui
-          </h2>
+  const onSubmit = (todo: TodoWithUser) => {
+    setTodos((prevState) => [...prevState, todo]);
+  };
 
-          <a className="UserInfo" href="mailto:Julianne.OConner@kory.org">
-            Patricia Lebsack
-          </a>
-        </article>
-      </section>
+  return (
+    <div className="App">
+      <FormAddTodo
+        users={usersFromServer}
```

Comment:
Can we just import this `usersFromServer` in this component instead of passing it through props?